docs/commercial/single-sign-on.rst

```diff
@@ -0,0 +1,90 @@
+Single Sign-On
+==============
+
+.. note::
+
+   This feature only exists on `Read the Docs for Business <https://readthedocs.com/>`__.
+
+
+Single Sign-On is supported on |com_brand| for Pro and Enterprise plans.
+:abbr:`SSO (Single Sign-On)` will allow you to grant permissions to your organization's projects in an easy way.
+
+Currently, we support two different types of Single Sign-On:
+
+* Authentication *and* authorization are managed by the Identity Provider (e.g. GitHub, Bitbucket or GitLab)
+* Authentication is managed by the Identity Provider (e.g. a ``@company.com`` verified email address)
+
+.. note::
+
+   SSO is currently in **Beta** and only GitHub is supported for now.
+   If you would like to apply for the Beta, please `contact us <mailto:support@readthedocs.com>`_.
+
+.. contents::
+   :local:
+   :depth: 2
+
+
+SSO with GitHub, Bitbucket or GitLab
+------------------------------------
+
+Using an Identity Provider that supports authentication and authorization allows you to manage
+"who have access to what projects on Read the Docs" directly from the provider itself.
+In case you want a user to have access to your documentation project under Read the Docs,
+that user just needs to be granted permissions in the GitHub, Bitbucket or GitLab repository associated with it.
+
+Note the users created under Read the Docs must have their GitHub, Bitbucket or GitLab
```

Comment:
It's fine using "VCS provider" as a general concept here I think. We use "VCS account" already in other documents, https://docs.readthedocs.io/en/stable/connected-accounts.html. Probably better to stick with that term.

I think it makes sense to mention them explicitly at the beginning maybe and use the general term in the rest of the document. It's good to avoid confusions with projects imported from outside these three providers. However, they have to be imported _manually_ which is not discoverable in .com

Reply:
I made some small changes for this. I left it explicit where I thought it makes sense to make it explicit and used "VCS social provider" or "VCS provider" as well.